apps/main/src/layout/default/index.tsx

```diff
@@ -59,10 +56,10 @@ const BaseLayout = ({ children }: { children: React.ReactNode }) => {
         maintenanceMessage={maintenanceMessage}
         handleNetworkChange={handleNetworkChange}
       />
-      <Container className={isMdUp ? 'hasFooter' : ''} globalAlertHeight={layoutHeight?.globalAlert}>
+      <Container globalAlertHeight={layoutHeight?.globalAlert}>
```

Comment:
so much duplication :sob: 
these index files are almost the same (nothing related to this PR)